index.js

```diff
@@ -120,7 +128,11 @@ async function main () {
     }
 
     const { area_keys, area_names: focus_areas } = get_focus_areas()
+
+    console.log('Writing site/scores.json')
     write_json_file(
         './site/scores.json', { area_keys, focus_areas, scores })
+
+    console.log('done')
```

Comment:
```suggestion
    console.log('Done')
```

Just to be consistent with the the other log lines.